rero_ils/modules/items/api/api.py

```diff
@@ -104,6 +104,9 @@ def reasons_not_to_delete(self):
         links = self.get_links_to_me()
         if links:
             cannot_delete['links'] = links
+        if self.item_record_type == 'issue' \
```

Comment:
so you want to prevent the deletion of the `deleted_issues`?

Reply:
It's used to disable the "trash" delete button when an issue has already the deleted status

Reply:
by doing this, we will not be able to delete an `issue` with a `deleted` status. if it is not the case, ignore my message.

Reply:
I think we could *never* delete a received issue, even if the status is deleted.

Reply:
hmmm... a good question for the PO.